NFIQ2/NFIQ2Algorithm/include/nfiq2_version.hpp

```diff
@@ -4,22 +4,31 @@
 #include <string>
 
 namespace NFIQ2 { namespace Version {
+
+/** Major version number. */
 extern const unsigned int Major;
+/** Minor version number. */
 extern const unsigned int Minor;
+/** Patch version number. */
 extern const unsigned int Patch;
 
+/** Full version information. */
 extern const std::string Full;
+/** Pretty version information. */
 extern const std::string Pretty;
+/** Build version status. */
 extern const std::string Status;
+/** Current build date. */
 extern const std::string BuildDate;
+/** Current build commit. */
 extern const std::string Commit;
 
-/*
- * Third-Party Components
- */
-
+/* Third-Party Components. */
```

Comment:
Leave as-is. This is a BSD-style thing for important comments/sections.

```suggestion
/* 
 * Third-Party Components. 
 */
```